portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,13 @@
  */
 function addRandomGreeting() {
   const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+      ['Run!',
```

Comment:
Style nitpick: When using a block-style array, put a break after the opening brace, like:

const my_arr = [
  'A',
  'B',
  'C',
];

For Google internal code, there would probably be a linter test that would require this.

Reply:
Thank you for letting me know! I'll fix it and take a look at the style guides to keep an eye out for that in the future.